From c246a87ef4fec68140a0fbabf2e3798bc4ab329d Mon Sep 17 00:00:00 2001
From: Artur Wieczorek <artwik@wp.pl>
Date: Sat, 21 Mar 2015 21:04:41 +0100
Subject: [PATCH] Add some Visual Studio specific files to the list of ignored
 files.

Following local files:
VS 2010 user specific settings (*.vcxproj.user )
VS 2012+ solution user options (*.suo)
should be ignored in commits by default.
---
 .gitignore | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/.gitignore b/.gitignore
index 1b77e38..3a41ae9 100644
--- a/.gitignore
+++ b/.gitignore
@@ -46,6 +46,7 @@
 /build/msw/*c*_mswhdll
 /build/msw/*c*_mswuhdll
 /build/msw/*.vcproj.*.user
+/build/msw/*.vcxproj.user
 /build/msw/vc*_mswud
 /build/msw/wx_vc8arm_*.vcproj
 /build/msw/wx_vc8arm.sln
@@ -56,6 +57,7 @@
 /build/msw/ipch
 /build/msw/wx_vc12.opensdf
 /build/msw/wx_vc12.sdf
+/build/msw/*.suo
 
 # /build/wince/
 /build/wince/vc_msw
-- 
1.9.5.msysgit.0

